src/export_command_templates.ts

```diff
@@ -59,7 +59,6 @@ export default {
     type: 'pandoc',
     arguments:
       '-f markdown --resource-path="${currentDir}" --resource-path="${attachmentFolderPath}" --lua-filter="${luaDir}/pdf.lua" --embed-resources --standalone --metadata title="${currentFileName}" -s -o "${outputPath}" -t pdf',
-    customArguments: '--pdf-engine=typst',
```

Comment:
@mokeyish Why is this in the custom arguments for PDF?

Reply:
But why would `--pdf-engine=typst` be the default?

Reply:
You can remove it.